src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -87,6 +87,8 @@ GraphingCalculator::GraphingCalculator()
     virtualKey->Key = (VirtualKey)187; // OemAdd key
     virtualKey->Modifiers = VirtualKeyModifiers::Control;
     ZoomInButton->KeyboardAccelerators->Append(virtualKey);
+
+    SharedShadow->Receivers->Append(GraphingControl);
```

Comment:
This code will crash on older version of windows

Reply:
This code shouldn’t run on older versions since it’s Graphing Calc but it’s probably good to add an explicit check anyway.